ProjectTraning/Task1/Driver.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ProjectTraning.Task1
+{
+    class Driver: Worker
```

Comment:
Missed access modifiers for all classes.
Seems you completely forgot about that.